modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -5,26 +5,29 @@
 [id="cluster-logging-deploy-multitenant_{context}"]
 = Installing OpenShift Logging into a multitenant network
 
-If you are deploying OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+In a cluster, if the _OpenShift SDN network isolation mode_ is configured to use either the _multitenant_ or _Network policy_ mode, projects in that cluster are isolated from each other: network traffic is not allowed between pods or services in different projects. Network policy is the default mode in {product-title} {product-version}.
```

Comment:
@jboxman Am I understanding this correctly: If Network policy is the default mode in {product-title} {product-version}, it seems very likely that users must perform at least one of the following steps in most deployments. If so, it seems like I should put this task, "Installing OpenShift Logging into a multitenant network," in the main flow of deployment tasks. WDYT?

Reply:
@jboxman Yes, that's unclear to me. 

@kabirbhartiRH, what do you think? It seems like most Logging deployments are going to be in clusters that have the OpenShift SDN or OVN-Kubernetes providers, right? 

@jboxman If we look at the TOC at [the top of the assembly](https://deploy-preview-33202--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-deploying?utm_source=github&utm_campaign=bot_dp#cluster-logging-deploy-multitenant_cluster-logging-deploying), we see that the two "Web Console" and "CLI" workflows merge back into a single set of "Post-intallation tasks". These post-installation tasks aren't optional or less important.  Ideally, I'd like to avoid this type of "funneling" and have two separate assemblies for each workflow, one the "Web Console" workflow and another for the "CLI" workflow. But I don't want to get into re-architecting now.

Reply:
Thanks you both. I just pushed changes which reflect that network isolation is probable, rather than merely possible.